job-service/model/src/main/java/org/verapdf/webapp/jobservice/model/entity/enums/Profile.java

```diff
@@ -1,6 +1,7 @@
 package org.verapdf.webapp.jobservice.model.entity.enums;
 
 public enum Profile {
+	WCAG_2_1_COMPLETE("WCAG 2.1 Complete"),
```

Comment:
move this value right after the simple wcag here

Reply:
"Complete" should be the first profile, since we select the first one as default on frontend and we don't want to change that there.